bitnami/minio/values-production.yaml

```diff
@@ -42,7 +42,7 @@ image:
 clientImage:
   registry: docker.io
   repository: bitnami/minio-client
-  tag: latest
+  tag: 2019.5.23-debian-9-r5
```

Comment:
the version is different for the server & client 5.14 vs 5.23

Reply:
Yes, it is because MinIO releases are failing in the pipeline but MinioClient is working fine, that is the reason.
At this moment we are working on a fix for MinIO, once fixed in the pipeline a new release will be triggered and both versions should be the same.
Although this change doesn't affect, I mean, `latest` is pointing to  `2019.5.23-debian-9-r5` in the client; so everything is in the same state than previously.